docs/usage/getting-started/private-packages.md

```diff
@@ -398,6 +399,14 @@ npmRegistries:
     npmAuthToken: <Decrypted PAT Token>
 ```
 
+Thus, to allow for private registries to be used locally via the `NPM_TOKEN` environment variable AND with renovate, your local `.yarnrc.yml` should contain
```

Comment:
```suggestion
Thus, to allow for private registries to be used locally via the `NPM_TOKEN` environment variable _and_ with Renovate, your local `.yarnrc.yml` must contain:

```

Changes:

- Use _emphasis_ styling, instead of CAPSLOCK text
- Capitalize proper nouns or brandnames, like Renovate
- Lead-in text for a codeblock usually needs to end with a `:`
- If a user must do something, for things to work properly, we use the keyword must
- Use a newline between the lead-in text and the codeblock to match Prettier linter formatting